### ISSUE TITLE
Is there a way to prevent Broadcasting Event from monitoring?

### ISSUE THREAD

**Issue by darron1217** [romanzipp/Laravel-Queue-Monitor#5]
Every broadcasting events are monitored by this package

How can I make exclude event from being monitored?

my event looks like this
```php
<?php

namespace Modules\Pottophoto\Events;

use Illuminate\Queue\SerializesModels;
use Modules\Pottophoto\Entities\Project;
use Illuminate\Contracts\Broadcasting\ShouldBroadcast;

class ProjectRenderingProgress implements ShouldBroadcast
{
    use SerializesModels;

    public $project;
    public $progress;

    /**
     * Constructor
     *
     * @return void
     */
    public function __construct(Project $project, int $progress)
    {
        $this->project = $project;
        $this->progress = $progress;
    }

    /**
     * Get the channels the event should be broadcast on.
     *
     * @return array
     */
    public function broadcastOn()
    {
        return ['asgardcms.pottophoto.projects'];
    }
}

```
![image](https://user-images.githubusercontent.com/8064923/40091835-002b8e18-58f5-11e8-81e8-99ce95277c15.png)

**Comment by romanzipp** [comment]
I'll have a look in to that

**Comment by darron1217** [comment]
It may be a big change but...

How about setting monitoring disabled as a deafult?

There's lots of broadcasts from external packages in my project.
I cannot add `DontMonitor` trait on all the jobs.

What do you think of creating `EnableMonitor` trait to make it as a whitelist? (instead of current blacklist system)

**Comment by romanzipp** [comment]
Yep, this would be the cleanest solution. I'll make the `QueueMonitor` trait required for the monitoring to be enabled.

**Comment by romanzipp** [comment]
I've released version [1.2.0](https://github.com/romanzipp/Laravel-Queue-Monitor/releases/tag/1.2.0) since this is a breaking change.

**Comment by darron1217** [comment]
What a lightning speed 👍
